extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php

```diff
@@ -11,7 +11,9 @@ class ARecoveryModule {
 	public static function isEnabled() {
 		global $wgEnableUsingSourcePointProxyForCSS;
 
-		return !empty( $wgEnableUsingSourcePointProxyForCSS );
+		$wgGlobalEnableSourcePoint = WikiFactory::getVarValueByName( 'wgGlobalEnableSourcePoint', Wikia::COMMUNITY_WIKI_ID );
+
+		return !empty( $wgEnableUsingSourcePointProxyForCSS ) || !empty( $wgGlobalEnableSourcePoint );
```

Comment:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/AdEngine2ContextService.class.php#L53

Does it mean we will have this value cached in DOM? How is AdEngine going to work without recovery but with `ads.context.opts.sourcePointRecovery=true`?